kafka-companion-frontend/src/app/no-data-placeholder/no-data-placeholder.component.html

```diff
@@ -1 +1 @@
-<p>No data to display</p>
+<p *ngIf="!progressBarService.progressSub.getValue()">No data to display</p>
```

Comment:
To zadziała? Próbujesz użyć prywatnego serwisu. Ja bym przeniósł ten warunek to gettera w pliku .ts

Reply:
zgadza się, leci błąd error TS2341: Property 'progressBarService' is private and only accessible within class 'NoDataPlaceholderComponent'.